contentcuration/contentcuration/static/js/edit_channel/uploader/views.js

```diff
@@ -440,6 +442,21 @@ var EditMetadataEditor = BaseViews.BaseView.extend({
       }
     }
   },
+  display_license_description: function(license_id){
```

Comment:
Set license_description text/display properties if content has the `Special Permissions` license